static/app/views/organizationGroupDetails/groupActivityItem.tsx

```diff
@@ -140,6 +142,63 @@ function GroupActivityItem({activity, orgSlug, projectId, author}: Props) {
               author,
             });
       case GroupActivityType.SET_RESOLVED_IN_COMMIT:
+        const deployedReleases: Array<BaseRelease> = [];
+        for (const release of activity.data.commit?.releases) {
+          if (release.dateReleased !== null) {
+            deployedReleases.push(release);
+          }
+        }
```

Comment:
Consider using `Array.filter` instead of the for loop:
```
const deployedReleases = (activity.data.commit?.releases || [])
          .filter(r => r.dateReleased !== null)
          .sort(
            (a, b) => moment(a.dateReleased).valueOf() - moment(b.dateReleased).valueOf()
          );
```